roles/vault/tasks/bootstrap.yml

```diff
@@ -25,7 +25,7 @@
   changed_when: no
   when: "'Vault is already initialized' not in vault_init_output.stdout"
   set_fact:
-    vault_token: "{{ (vault_init_output.stdout|from_json)['root_token'] }}"
+    vault_root_token: "{{ (vault_init_output.stdout|from_json)['root_token'] }}"
```

Comment:
`when: "{{ vault_root_token is defined }}"` won't work. An older run's `vault_root_token` can still be in `security.yml`

Maybe
```yaml
- set_fact:
    vault_token: "{{ vault_root_token }}"
  when: "'Vault is already initialized' in vault_init_output.stdout"
```
so `vault_token` gets set if vault is already initialized. I think we can assume that if vault is already initialized that `vault_root_token` is valid.